index.html

```diff
@@ -30,14 +30,74 @@ <h1 class="info_box">
             <li><a href="#"><img src="two.svg" width="1" height="15" alt="two"></a></li>
-         </ul>
-    <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+        </ul>
+        <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+    </section>
+
+        
+    <section class="section_works_container">
+       <header>
+            <img class="sec_in" src="snap.svg" alt="selfie">
+            <div class="tonic">
+                <h2>Tonic</h2>
+            </div>
+            <div>
+                <ul class="sec_h">
+                    <li><a href="#"><img src="canopy.svg" alt="canopy"></a></li>
+                    <li><a href="#"><img src="Count.svg" width="1" height="5" alt="count"></a></li>
+                    <li><a href="#"><img src="bed.svg" height="22" alt="bed"></a></li>
+                    <li><a href="#"><img src="cc.svg" width="1" height="5" alt="dot"></a></li>
+                    <li><a href="#"><img src="time.svg" alt="date"></a></li>
+                </ul>
+            </div>
+       </header>
+        <section>
+            <h3 class="p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h3>
+            <div>
+                <ul class="push">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">CSS</a></li>
+                    <li><a href="#">Javascript</a></li>
+                </ul>
+            </div>
+            <h3 class="endpush"><a href="#">See Project</a></h3> 
+        </section>
+    </section>
+
+    <section class="about_section">
+        <h2 class="about_my">About Myself</h2>
+        <h4 class="sec_abt">
+            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </h4>
+            <p class="contact_section"><a href="#">Get my resume</a></p>
+            <ul>
+                <li><a href="#"><img src="cat.svg" width="1" height="15" alt="cat"></a></li>
+                <li><a href="#"><img src="bird.svg" width="1" height="15" alt="bird"></a></li>
+                <li><a href="#"><img src="m.svg" width="1" height="15" alt="M"></a></li>
+                <li><a href="#"><img src="n.png" width="1" height="15" alt="N"></a></li>
+                <li><a href="#"><img src="two.svg" width="1" height="15" alt="two"></a></li>
+            </ul>
+            <div>
+                <ul class="dropim">
+                    <li class="dropim"><a href="#">languages</a></li>
+                    <li class="dropim"><a href="#">Javascript</a></li>
+                    <li class="dropim"><a href="#">HTML</a></li>
+                    <li class="dropim"><a href="#">CSS</a></li>
+                    <li class="fr_sk">Frameworks</li>
+                    <li class="fr_sk">Skills</li>
+                </ul>
+            </div>
```

Comment:
The same comment goes here as the project section stick to the Figma design.

| Your Design | Figma Design |
|--------------|--------------|
| <img width="303" alt="image" src="https://user-images.githubusercontent.com/34578631/170740600-7cafb8f5-26dc-467c-9678-afdb7bf152ad.png"> | <img width="371" alt="image" src="https://user-images.githubusercontent.com/34578631/170740671-546c61ff-ca09-42e0-a539-bc609c75f748.png"> |